boot/bootutil/src/bootutil_misc.c

```diff
@@ -457,6 +453,28 @@ boot_write_magic(const struct flash_area *fap)
     return 0;
 }
 
+
+/**
+ * Write trailer data; status bytes, swap_size, etc
+ *
+ * @returns 0 on success, != 0 on error.
+ */
+int
+cy_boot_write_trailer(const struct flash_area *fap, uint32_t off,
```

Comment:
`cy_` prefixes are not acceptable here. Also what would happen if a reset happens while inside the `flash_area_write_byte` loop?

Reply:
yes, sure

I left it intentionally to highlight which API required to be implemented in different way

I'll change it to neutral as soon as we finalize the code itself